export-samples/html/cv-galactic-tech-innovations.html

```diff
@@ -0,0 +1,692 @@
+
+        /* colors */
+        --primary_color: #247a7e; /* only this color can be changed  */
+        --white_color: #f4f2f1;
+        --black_color: #0d1418;
+        --gray_color_600: #c6c3be;
+        --gray_color_800: #787673;
+        --gray_color_900: #5c5a58;
+      }
+      /* General styles */
+      * {
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+      }
+      html {
+        font-family: Roboto, sans-serif;
+        font-size: 16px;
+        color: var(--black_color);
+        font-style: normal;
+        background-color: var(--white_color);
+      }
+      body {
+        display: flex;
+        flex-direction: column;
+        min-height: 100vh;
+        padding: var(--spacing_64) var(--spacing_16);
+      }
+      .container {
+        display: grid;
+        grid-template-areas:
+          'header'
+          'aside'
+          'main';
+        row-gap: var(--spacing_32);
+        column-gap: var(--spacing_32);
+      }
+      @media (min-width: 728px) {
+        body {
+          padding: var(--spacing_64);
+        }
+        .container {
+          column-gap: var(--spacing_32);
+        }
+      }
+      @media (min-width: 1024px) {
+        .container {
+          grid-template-areas:
+            'header header'
+            'aside main'
+            'aside main';
+          max-width: 1220px;
+          align-self: center;
+        }
+      }
+
+      /* header styles */
+      header {
+        grid-area: header;
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        gap: var(--spacing_4);
+        align-self: stretch;
+      }
+      .header-title {
+        min-width: 100%;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        flex: auto;
+        /*justify-content: space-between;*/
+      }
+      .header-name {
+        display: inline-block;
+        /*min-width: fit-content;*/
+        font-family: Mohave;
+        font-size: 64px;
+        font-weight: 500;
+        letter-spacing: 4.8px;
+        text-transform: uppercase;
+        color: var(--primary_color);
+        padding: 0 var(--spacing_16) 0 0;
+      }
+      .header-left-gap {
+        display: none;
+      }
+      .header-right-gap {
+        display: block;
+        /*min-width: fit-content;*/
+        flex: auto;
+      }
+      .header-bordered-gap {
+        border-bottom: 1px solid var(--primary_color);
+      }
+      .header-subtitle {
+        display: flex;
+        flex-direction: column;
+        padding: 0 var(--spacing_16) var(--spacing_16) 0;
+      }
+      .header-surname {
+        font-family: Mohave;
+        font-size: 36px;
+        font-weight: 500;
+        letter-spacing: 2.4px;
+        text-transform: uppercase;
+        color: var(--gray_color_900);
+        white-space: nowrap;
+        padding-right: var(--spacing_16);
+      }
+      .header-job-position {
+        font-family: 'Raleway';
```

Comment:
We don't need it. Raleway font is by default

Reply:
OK